std_msgs/Header header
float64[] polynomial_durations
float64[] coefficients